models/user.go

```diff
@@ -31,3 +31,21 @@ func SaveUser(email string, password string) error {
 	}
 	return nil
 }
+
+func FindOneUserBy(condition interface{}) (User, error) {
```

Comment:
not sure if we need `One` 😅  if we are writing `User` in the singular that should indicate a single User in response already

Reply:
Make sense. Fixed in 5ce542d99b7688e89218339e206436fb78659361 1c6171adf5c87a98fda697317ead8a29783b77a5